python/huggingfaceserver/huggingfaceserver/model.py

```diff
@@ -110,6 +127,36 @@ def __init__(
         self.vllm_engine_args = engine_args
         self.use_vllm = not kwargs.get("disable_vllm", False) if _vllm else False
         self.ready = False
+        self.use_triton = kwargs.get("enable_triton", False) if _triton else False
+        if self.use_triton:
+            self.use_vllm = False
+            self.triton_server = None
+            self._triton_client_instance = None
+            self._triton_config = (
+                TritonConfig.from_dict(triton_config)
+                if triton_config
+                else TritonConfig(
+                    allow_http=False,
+                    http_port=8000,
+                    allow_grpc=True,
+                    grpc_port=8001,
+                    allow_metrics=True,
+                    allow_gpu_metrics=True,
+                    allow_cpu_metrics=True,
+                    metrics_port=8002,
+                    cache_config=["local,size=1048576"],
+                    exit_timeout_secs=5,
+                )
+            )
+
+    @property
+    def _triton_client(self):
+        if self._triton_client_instance is None:
+            url = f"grpc://localhost:{self._triton_config.grpc_port}"
+            self._triton_client_instance = AsyncioModelClient(
```

Comment:
how is the `AsyncioModelClient` different from the triton's grpc python client ?